util/pivnet_meta.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+slug=$1
```

Comment:
Can you add validation that the argument exists?

If it doesn't, can you output the possible values which are expected to be used for this project? (elastic-runtime, stackdriver-nozzle, etc)